x-pack/plugins/maps/server/register_integrations.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { CoreSetup } from 'kibana/server';
+import { CustomIntegrationsPluginSetup } from '../../../../src/plugins/custom_integrations/server';
+import { APP_ID } from '../common/constants';
+
+export function registerIntegrations(
+  core: CoreSetup,
+  customIntegrations: CustomIntegrationsPluginSetup
+) {
+  customIntegrations.registerCustomIntegration({
+    id: 'ingest_with_gdal',
+    title: i18n.translate('xpack.maps.registerIntegrations.gdal.integrationTitle', {
+      defaultMessage: 'Upload geo data with GDAL',
+    }),
+    description: i18n.translate('xpack.maps.registerIntegrations.gdal.integrationDescription', {
+      defaultMessage:
+        'Upload shapefiles or ingest from relational databases such as PostGIS and OracleSpatial',
```

Comment:
```suggestion
        'Upload shapefiles or ingest from relational databases, such as PostGIS and OracleSpatial.',
```

Reply:
Alternatively, I noticed that the descriptions all end with "with xxx".  Can the description be:

Upload shapefiles or ingest from relational databases with GDAL.

(or if there is room, I'd spell out GDAL)

Reply:
How about changing 'or' to 'and'?

Upload shapefiles and ingest from relational databases with GDAL.

Reply:
`OracleSpatial` should have a space between words, see for example this blog post

https://blogs.oracle.com/oraclespatial/post/oracle-spatial-studio-212-is-now-available